cellpack/autopack/writers/ImageWriter.py

```diff
@@ -0,0 +1,256 @@
+from aicsimageio.writers.ome_tiff_writer import OmeTiffWriter
+from pathlib import Path
+import numpy
+from scipy.ndimage import convolve
+
+"""
+ImageWriter provides a class to export cellpack packings as tiff images
+"""
+
+
+class ImageWriter:
```

Comment:
I would suggest that this class should be called Voxelizer or Voxelization etc.  Writing the image out is only a tiny part of what is in here!